server/data/hmppsAuthClient.ts

```diff
@@ -10,15 +10,14 @@ import RestClient from './restClient'
 const timeoutSpec = config.apis.hmppsAuth.timeout
 const hmppsAuthUrl = config.apis.hmppsAuth.url
 
-function getSystemClientTokenFromHmppsAuth(username?: string): Promise<superagent.Response> {
+function getSystemClientTokenFromHmppsAuth(): Promise<superagent.Response> {
```

Comment:
Removing support for getting system client tokens with a `username` because not applicable with GOV.UK One Login.